README.md

```diff
@@ -29,6 +29,9 @@ Description of Variables
 **P_ID**
 Identifier of the different products/articles.
 
+**C_ID**
```

Comment:
Don't include this as column in the created data set. It is just an internal data-processing step in the simulation that won't be visible in the observable data.

Reply:
I deleted the 'C_ID' column README.md